src/components/Participants/index.jsx

```diff
@@ -11,7 +11,7 @@ const Participants = ({ rawParticipants }) => {
   const participants = new Set();
   rawParticipants.map(({ node: { data: { Github_Username: username } } }) => {
     // dedupe
-    participants.add(username ? username.toLowerCase() : 'react-knowledgeable');
+    return participants.add(username ? username.toLowerCase() : 'react-knowledgeable');
```

Comment:
I actually fixed this in the GH RSVP branch by making it a forEach. Since we aren’t using the returned array a map is not necessary. Just a nit though.

Reply:
sure, i just ocd abt linter issues, feel free to ignore me :p

Reply:
I meant we could do this instead (I'm on board with addressing the linter issues :P):

```suggestion
  rawParticipants.forEach((...) => {
    participants.add(username ? username.toLowerCase() : 'react-knowledgeable');
  })
```

Reply:
@thchia so use ur suggestion but clean up the lines?